include/picongpu/particles/startPosition/generic/Free.hpp

```diff
@@ -79,10 +79,7 @@ namespace picongpu
                     using Functor = T_Functor;
 
                     template<typename T_SpeciesType>
-                    struct apply
-                    {
-                        using type = Free;
-                    };
+                    using fn = Free;
```

Comment:
Because `fn` is mp11's terminology. See: https://www.boost.org/doc/libs/master/libs/mp11/doc/html/mp11.html:

> A quoted metafunction is a class with a public metafunction member called fn, for example ...